source/includes/_endpoint-dataset.md

```diff
@@ -220,6 +229,13 @@ dataset_projects  | List of Strings | Project IDs having read-access to the data
 GET /datasets/search/?q={query}&f={filter}&limit={limit}&offset={offset}&group_variables_list={group_variables_list}&projection={projection}  HTTP/1.1
 ```
 
+```
+import pycrunch
+site = pycrunch.connect("me@mycompany.com", "yourpassword", "https://app.crunch.io/api/")
+results = site.datasets.follow('search', 'q=findme&embedded_variables=True').value
```

Comment:
The pycrunch example still points to `/datasets/X/search`, we probably want to update it to `/search` to avoid users confusion.